examples/server.toml

```diff
@@ -50,48 +68,44 @@ tls_key = "/data/key.pem"
 domain = "idm.example.com"
 #
-#   The origin for webauthn. This is the url to the server, with the port included if
-#   it is non-standard (any port except 443)
-# origin = "https://idm.example.com"
+#   The origin for webauthn. This is the url to the server,
+#   with the port included if
+#   it is non-standard (any port except 443). This must match
+#   or be a descendent of the
+#   domain name you configure above. If these two items are
+#   not consistent, the server WILL refuse to start!
+#   origin = "https://idm.example.com"
 origin = "https://idm.example.com:8443"
 #
-#   The role of this server. This affects features available and how replication may interact.
+#   The role of this server. This affects available features
+#   and how replication may interact.
 #   Valid roles are:
 #   - WriteReplica
-#     This server provides all functionality of Kanidm. It allows authentication, writes, and
+#     This server provides all functionality of Kanidm. It
+#     allows authentication, writes, and
 #     the web user interface to be served.
 #   - WriteReplicaNoUI
-#     This server is the same as a write_replica, but does NOT offer the web user interface.
+#     This server is the same as a WriteReplica, but does NOT
+#     offer the web user interface.
 #   - ReadOnlyReplica
-#     This server will not writes initiated by clients. It supports authentication and reads,
-#     and must have a replication agreement as a source of it's data.
+#     This server will not writes initiated by clients. It
+#     supports authentication and reads,
+#     and must have a replication agreement as a source of
+#     its data.
 #   Defaults to "WriteReplica".
 # role = "WriteReplica"
 #
-# This section if uncommented will enable online - automatic backups of your database.
-# [online_backup]
-#
+[online_backup]
 #   The path to the output folder for online backups
-#   Defaults to "" (no path set)
-# path = "/var/lib/kanidm/backups/"
-#
-#   The schedule to run online backups. All times are interpreted in UTC.
-#   The format of the cron expression is:
-#
+path = "/data/kanidm/backups/"
+#   The schedule to run online backups (see https://crontab.guru/)
+#   every day at 22:00 UTC (default)
+schedule = "00 22 * * *"
+#    four times a day at 3 minutes past the hour, every 6th hours
+# schedule = "03 */6 * * *"
+#   We also support non standard cron syntax, with the following format:
 #   sec  min   hour   day of month   month   day of week   year
-#
-#   - to run a 6:09 pm every day.
-#   "0    9    6      *              *       *             *   "
-#   "0 9 6 * * * *"
-#
-#   - to run at midnight daily
-#   @daily
-#
-#   - to run every hour
-#   @hourly
-#
-#   Defaults to "@daily"
-# schedule = "@daily"
+#   (it's very similar to the standard cron syntax, it just allows to specify the seconds
+#   at the beginning and the year at the end)
 #   Number of backups to keep (default 7)
-# versions = 7
-
+versions = 7
```

Comment:
This didn't need to be uncommented if it's the default 😄

Reply:
Oh true. I'll fix that.